docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -31,12 +32,15 @@ image::images/cases-ui-connector.png[]
 * *{sn}*: To send cases to {sn}
 * *{jira}*: To send cases to {jira} or Jira Service Desk
 * *{ibm-r}*: To send cases to {ibm-r}
+* *{swimlane}*: To send cases to {swimlane}
 
 . Fill in the following:
 * *Connector name*: A name for the connector.
 * *URL*: The URL of the external system to which you want to send cases.
+* *API Url* (Swimlane connectors only): The URL of the Swimlane instance to which you want to send cases.
```

Comment:
Should "Swimlane" be "{swimlane}" here?
Also, URL instead of Url?
```suggestion
* *API URL* ({swimlane} connectors only): The URL of the {swimlane} instance to which you want to send cases.
```

Reply:
Checked UI text for the Swimlane connector fields and the URL was written as Url.